ansible/redpanda/cluster/roles/demo_certs/tasks/create-ca.yml

```diff
@@ -8,7 +8,7 @@
   ansible.builtin.file:
     path: "{{ root_ca_dir }}"
     state: directory
-    mode: "0600"
+    mode: "0755"
```

Comment:
can you help me understand the opening up of permissions here? should this be 0700 since it's a directory?

Reply:
It breaks CI reliably without this change

Reply:
Source: I made this change in the other PR and it stopped breaking CI